integrations/access/slack/bot.go

```diff
@@ -41,6 +41,10 @@ import (
 )
 
 const slackMaxConns = 100
+
+// textObjectMaxCharLimit is the max length for a slack 'text' object.
+const textObjectMaxCharLimit = 3000
```

Comment:
Can we please add a link to slack api where the limit is mentioned? If there's one.